integration-tests/src/test/java/oracle/weblogic/domain/FluentdSpecification.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import javax.annotation.Nullable;
+
+import io.kubernetes.client.openapi.models.V1EnvVar;
+import io.kubernetes.client.openapi.models.V1ResourceRequirements;
+import io.kubernetes.client.openapi.models.V1VolumeMount;
+import io.swagger.annotations.ApiModelProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+
+public class FluentdSpecification {
+
+  @ApiModelProperty("The fluentd configuration text, specify your own custom fluentd configuration.")
+  private String fluentdConfiguration;
+
+  /**
+   * The Fluentd sidecar image.
+  */
+  @ApiModelProperty(
+          "The Fluentd container image name. Defaults to ")
```

Comment:
What does it default to? :)

Reply:
This is for the integration test class, removed `Defaults to`